components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java

```diff
@@ -200,8 +200,8 @@ public static void createApplication(String appName, String appOwner, String app
 
             AuthenticationStep authenticationStep2 = new AuthenticationStep();
             LocalAuthenticatorConfig basic = new LocalAuthenticatorConfig();
-            basic.setName("BasicAuthenticator");
-            basic.setDisplayName("basic");
+            basic.setName(FrameworkConstants.BASIC_AUTHENTICATOR_CLASS);
```

Comment:
The constant taken from - https://github.com/wso2/carbon-identity-framework/blob/master/components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java#L88